### ISSUE TITLE
<xsd:choice> <xsd:sequence> <xsd:element> not making it into request

### ISSUE THREAD

**Issue by s-j-h** [mvantellingen/python-zeep#150]
As email

elements in a choice -> sequence (marked with #! below) aren't coming out after being serialised, tried using the _value_N syntax & normal dict creation, example below

```
import datetime
from zeep import Client
client = Client('https://secure.contego.com/wsdl/v1/contegoRequesterV1.wsdl')
creds = {
    "OrganisationUID":"111",
    "MD5signature":"222",
    "ProfileUID":"333",
}
header = {
    "TransactionRef":"1234"
}
check_person_data_type =  client.get_type("ns1:CheckPersonData")
check_person_data = check_person_data_type(
    Title="Mr",
    Forename="Joe",       #!
    Middlename="Foo",    #!
    Surname="Bloggs",        #!
    Gender="MALE",
    DOB=datetime.date(year=1970,month=1,day=1),
    CountryOfResidence= "ZW"
)
print client.service.Contego(
    Credentials = creds,
    Header = header,
    CheckPerson = check_person_data
)
```

snippet from wsdl

```
<xsd:complexType name="CheckPersonData">
<xsd:sequence>
<xsd:element minOccurs="0" name="PersonType">...</xsd:element>
<xsd:choice>
<xsd:sequence>
<xsd:element minOccurs="0" name="Forename" type="xsd:string"/>
<xsd:element minOccurs="0" name="Surname" type="xsd:string"/>
<xsd:element minOccurs="0" name="Maidenname" type="xsd:string"/>
<xsd:element minOccurs="0" name="Middlename" type="xsd:string"/>
</xsd:sequence>
</xsd:choice>
<xsd:element minOccurs="0" name="Gender">
...etc
```

**Comment by mvantellingen** [comment]
Thanks, should be fixed, let me know otherwise!

**Comment by s-j-h** [comment]
Hiya - just got round to giving this a go, output of the below still lacks forename/surname/maidenname/middlename in the request

The only substantial difference I can see from the wsdl below & the new unit test in 3b60197 is I am .get_type()'ing a named `xsd:complex-type` rather than an `xsd:element`

```
import datetime
from zeep import Client
client = Client('https://secure.contego.com/wsdl/v1/contegoRequesterV1.wsdl')
creds = {
    "OrganisationUID":"111",
    "MD5signature":"222",
    "ProfileUID":"333",
}
header = {
    "TransactionRef":"1234"
}
import logging.config

logging.config.dictConfig({
    'version': 1,
    'formatters': {
        'verbose': {
            'format': '%(name)s: %(message)s'
        }
    },
    'handlers': {
        'console': {
            'level': 'DEBUG',
            'class': 'logging.StreamHandler',
            'formatter': 'verbose',
        },
    },
    'loggers': {
        'zeep.transports': {
            'level': 'DEBUG',
            'propagate': True,
            'handlers': ['console'],
        },
    }
})

check_person_data_type =  client.get_type("ns1:CheckPersonData")
check_person_data = check_person_data_type(
    Title="Mr",
    Forename="Joe",       #!
    Middlename="Foo",    #!
    Surname="Bloggs",        #!
    Gender="MALE",
    DOB=datetime.date(year=1970,month=1,day=1),
    CountryOfResidence= "ZW"
)
print client.service.Contego(
    Credentials = creds,
    Header = header,
    CheckPerson = check_person_data
)
```

**Comment by mvantellingen** [comment]
Finally reproduced the issue and should now be fixed on master (next 0.16 release)

**Comment by s-j-h** [comment]
Request correctly generated with 0b2a43fc4ec79eb4be35a51ae3104d285219f010

Thanks!
